packages/noports_core/lib/src/common/ssh_key_utils.dart

```diff
@@ -0,0 +1,217 @@
+import 'dart:async';
```

Comment:
All SSH Key Management is deferred to these classes. The AtSSHKeyPair takes a few extra ms (5ms in my tests) to parse a privateKey, but the benefit is that we have a common interface.

Reply:
Public Key encoding typically takes 3ms